cmd/build.go

```diff
@@ -183,3 +237,65 @@ func findDockerRootDir(workingDir, buildPath string) (string, error) {
 		searchPath = dir
 	}
 }
+
+func createDockerExecutor(shell exec.Executor, contextRootDir string) *kaniko.DockerExecutor {
+	dockerCfg := kaniko.ContainerConfig{
+		Image: "gcr.io/kaniko-project/executor:v1.7.0",
```

Comment:
We talked about making this image variable, do we plan to do it in an other PR ?

Reply:
Yes, when we can use a config file to put all this new configuration.

Otherwise we'd have to set a command line flag for each individual setting, I think it would make the cli a bit too hard to use...